next/components/article-teaser.tsx

```diff
@@ -36,6 +36,7 @@ export function ArticleTeaser({ article }: ArticleTeaserProps) {
       <Link
         href={article.path.alias}
         className="absolute inset-0 cursor-pointer"
+        aria-label={`${t("article-link")}${" | "}${article.title}`}
       />
```

Comment:
I'm wondering if we need the "Article link | " prefix here? Screen readers etc would announce it as a link so maybe it's redundant.

Idea: This link could instead have aria-labelledby={titleId}, which we can pass to the h3 on line 20? Rather than duplicating info in the label again

Reply:
I restructured this so that the link would be the wrapping element and therefore no need for aria labelling.